### DISCUSSION TITLE
Finally built one!

### DISCUSSION THREAD

**Discussion by andypiper** [martinwoodward/octolamp#28] [Show and tell]
This is a fantastic build, Martin, thank you - lovely attention to detail in the case design.

I found some ESP8266 D1 Mini modules that have USB-C connections and since that is thankfully becoming more common than micro-USB, I used one of those instead. Only a touch pricier. ([Amazon UK](https://amzn.to/3YAqds1))

![IMG_0036](https://github.com/martinwoodward/octolamp/assets/552452/58865c11-de10-4cdf-8a18-4cbf13ed219a)
![IMG_0037](https://github.com/martinwoodward/octolamp/assets/552452/3d104af3-63f9-45b9-be41-d20214611fe6)
![IMG_0039](https://github.com/martinwoodward/octolamp/assets/552452/1bf51bfe-f54e-462b-b310-b4e4f2f4fb25)
![IMG_0043](https://github.com/martinwoodward/octolamp/assets/552452/c903ea62-39d0-4b1a-bf06-255e65561a78)

Next step is to do some fun light sync with other WLED devices around my studio space!

**Comment by martinwoodward** [comment]
Great build!
